src/ChunkData.cpp

```diff
@@ -580,7 +584,11 @@ cChunkData::sChunkSection * cChunkData::Allocate(void)
 
 void cChunkData::Free(cChunkData::sChunkSection * a_Section)
 {
-	m_Pool.Free(a_Section);
+	if (a_Section != nullptr)
+	{
+		m_WorldMemoryCounter.DecrementApproximateChunkRAM(sizeof(cChunkData::sChunkSection));
+		m_Pool.Free(a_Section);
```

Comment:
No, but if you are going to make the comparison anyway, then you might as well avoid the call on null sections.